pkg/stub/arbiter.go

```diff
@@ -0,0 +1,122 @@
+package stub
+
+import (
+	"fmt"
+	"github.com/Percona-Lab/percona-server-mongodb-operator/internal/mongod"
+	"github.com/Percona-Lab/percona-server-mongodb-operator/internal/util"
+	"github.com/Percona-Lab/percona-server-mongodb-operator/pkg/apis/psmdb/v1alpha1"
+	"github.com/sirupsen/logrus"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (h *Handler) ensureReplsetArbiter(m *v1alpha1.PerconaServerMongoDB, replset *v1alpha1.ReplsetSpec, resources corev1.ResourceRequirements) (*appsv1.StatefulSet, error) {
+	if replset.Arbiter == nil {
+		replset.Arbiter = &v1alpha1.Arbiter{
+			Enabled: false,
+			Size:    0,
+		}
+	}
+	if replset.Arbiter.Enabled && replset.Arbiter.Size == 0 {
```

Comment:
@heartwilltell we need to have logic that ensures:

1. The operator does not create a "majority" of arbiters. The math for this is:
    ```
    math.Ceil(float64(replsetSize) * 0.50)
    ```
    Example: 3-node replset max arbiter count is 1, 5-node max is 2, etc.
1. Does not cannot create more than 6 arbiters, due to MongoDB 7 vote maximum.

Reply:
Great idea. I will implement it.